instrumentation/oshi/library/build.gradle.kts

```diff
@@ -3,11 +3,9 @@ plugins {
   id("com.google.osdetector")
 }
 
-// 5.5.0 is the first version that works on arm mac
-val oshiVersion = if (osdetector.os == "osx" && osdetector.arch == "aarch_64") "5.5.0" else "5.3.1"
-
 dependencies {
-  library("com.github.oshi:oshi-core:$oshiVersion")
+  // 5.5.0 is the first version that works on arm mac
+  library("com.github.oshi:oshi-core:5.5.0")
```

Comment:
if your goal is to remove the variable from library declaration then you could also
```
if (osdetector.os == "osx" && osdetector.arch == "aarch_64" && !(findProperty("testLatestDeps") as Boolean)) {
  // 5.5.0 is the first version that works on arm mac
  configurations.testRuntimeClasspath.get().resolutionStrategy.force("com.github.oshi:oshi-core:5.5.0")
}
```